tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js

```diff
@@ -124,14 +119,23 @@ context('OpenCV. Intelligent scissors. Histogram Equalization.', () => {
             cy.get('body').type('{Ctrl}'); // Checking hotkey
             cy.get('.cvat_canvas_threshold').should('exist');
             cy.get('body').type('{Esc}'); // Cancel drawing
+            cy.get('.cvat-opencv-control-popover-visible').should('not.exist');
+            cy.get('.cvat-opencv-control-popover').should('be.hidden');
+            cy.get('.cvat-opencv-control-popover').should('have.attr', 'style').and('include', 'pointer-events: none');
```

Comment:
I am not sure we should test that the popovers are closed now.
At least check for ``pointer-events`` seems to be extra.

Reply:
Without this check, there is a high probability of getting error ```AssertionError: Timed out retrying after 25000ms: Expected to find element: `.*-popover-visible`, but never found it.```

Removed heck for ```pointer-events```.